libcontainer/utils/utils.go

```diff
@@ -113,3 +115,38 @@ func Annotations(labels []string) (bundle string, userAnnotations map[string]str
 	}
 	return
 }
+
+func GetSystemCPUCores() (cpuNum uint32, _ error) {
```

Comment:
```suggestion
func SystemCPUCores() (cpuNum uint32, _ error) {
```

Also needs a comment to explain how this differs from https://pkg.go.dev/runtime#NumCPU